src/assets/images/tco19_logo_black.svg

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<svg width="109px" height="81px" viewBox="0 0 109 81" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+<svg width="109px" height="81px" viewBox="0 0 109 81" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" role="img" aria-label="Topcoder Open 2019">
```

Comment:
`role="img" aria-label="Topcoder Open 2019"`

@afrisalyp You are still using full form instead of "tco19" which is short form.